### ISSUE TITLE
Development process and documentation

### ISSUE THREAD

**Issue by jessykate** [cobudget/cobudget#109]
- [x] Document deployment process
- [x] database backups
- [x] ~Set up travis for UI~ AFAICT there are no tests for the frontend, so this is moot. we should make some (made another ticket for this).
- [x] ~Consolidate cobudget repos under greaterthan in github~ waiting on this for a bit.
- [x] Move tickets to github
- [x] Clean up waffle.io
- [ ] Mine through [old cobudget sprint board](https://trello.com/b/XIXOlhmX/old-cobudget-sprint-board-copy) for relevant tickets
- [x] migrate tickets from [this spreadsheet](https://docs.google.com/spreadsheets/d/1nYvOnd6gDBO1QR9YdmXOKpSER5-SCCQRo6YITU5d_xE/edit#gid=0)

**Comment by jessykate** [comment]
got through almost everything but the old sprint board! there's a _lot_ there :).

**Comment by chime-mu** [comment]
I've added some suggestions to slack:#process that might influence this cleanup. But potentially this should be done as  new ticket. What your take on this @jessykate ?

**Comment by jessykate** [comment]
yes i agree. i would consider _this_ ticket as done, as the intention was to set in place our basic processes and capture the initial documentation. there will always be more dev/process work to do of course. i'll make a separate ticket for backlog cleanup and include the old cobudget sprint board.
